### DISCUSSION TITLE
NEW

### DISCUSSION THREAD

**Discussion by ghost** [zippity21/Stardew_Valley_Regression_Mod#38] [Q&A] [answered]
Is anyone able to explain what NoFriendshipPenalty entails? 
	"NoFriendshipPenalty": true,
	
Oh yeah, are there any consequences for blowouts? How do you clean your clothes if they get dirty?

**Comment by zippity21** [comment, marked as answer]
This system still needs to be reworked a little, but the idea is that if you wet/mess near people, they may notice it would affect your relationship. The radius  of detection and magnitude of change are dependant on a number of things. But to your point regarding blowouts; they cause your pants to become dirty (not just your underwear) so it's easier for people to notice (larger area of detection).

Setting NoFriendshipPenalty should make it so someone noticing your soiling won't change their relationship with you.

Currently, when you change into new underwear, you also change into new pants, so there is no need to treat them separately.

**Comment by Forestspiritfarmer** [comment]
tell me where to put that command in

**Reply by GardensStarlit** [reply]
Open up the Regression Mod folder wherever your mods are saved, and inside the folder just called "Regression", you'll find a JSON file named "config" - you can open it up in the notepad app, edit the line from "true" to "false" - then, save the file after you've changed it, and it should apply the next time you launch the game.
